### PR TITLE
method signatures

### PR DESCRIPTION
**Pull request by itkg-ahidri**

### CODE REVIEW

File: FrontBundle/Routing/DatabaseRouteLoader.php [hunk 1]
```diff
@@ -118,10 +118,10 @@ protected function orderNodes(array $nodes)
 
     /**
      * @param $language
-     * @param $site
+     * @param ReadSiteInterface $site
      * @return array
      */
-    protected function initializeNodes($language, $site)
+    protected function initializeNodes(array $language, ReadSiteInterface $site)
```
Comment (nicolasThal) [inline FrontBundle/Routing/DatabaseRouteLoader.php]: Language is not an array, only a string

File: FrontBundle/Routing/DatabaseRouteLoader.php [hunk 2]
```diff
@@ -118,10 +118,10 @@ protected function orderNodes(array $nodes)
 
     /**
      * @param $language
```
Comment (nicolasThal) [inline FrontBundle/Routing/DatabaseRouteLoader.php]: you forgot to update this parameter

**Comment by itkg-ahidri** [comment]
@nicolasThal  done

File: FrontBundle/Routing/DatabaseRouteLoader.php [hunk 3]
```diff
@@ -117,11 +117,11 @@ protected function orderNodes(array $nodes)
...
 
     /**
-     * @param $language
-     * @param $site
+     * @param string $language
+     * @param ReadSiteInterface $site
      * @return array
      */
-    protected function initializeNodes($language, $site)
+    protected function initializeNodes( $language, ReadSiteInterface $site)
```
Comment (nicolasThal) [inline FrontBundle/Routing/DatabaseRouteLoader.php]: there is still a blank space before the $language

**Comment by ngilain** [comment]
:+1:

File: FrontBundle/Routing/DatabaseRouteLoader.php [hunk 4]
```diff
@@ -117,11 +117,11 @@ protected function orderNodes(array $nodes)
     }
 
     /**
-     * @param $language
-     * @param $site
+     * @param string $language
```
Comment (nicolasThal) [inline FrontBundle/Routing/DatabaseRouteLoader.php:120]: you should also align the parmaeters

File: FrontBundle/Routing/DatabaseRouteLoader.php [hunk 5]
```diff
@@ -117,11 +117,11 @@ protected function orderNodes(array $nodes)
...
+     * @param ReadSiteInterface $site
```
Comment (nicolasThal) [inline FrontBundle/Routing/DatabaseRouteLoader.php:121]: and put a blank line between the @param and @return

**Pull request closed without merging**